Assets/Scripts/Auction/RandomisedAuctionStage.cs

```diff
@@ -0,0 +1,53 @@
+using CollectionExtensions;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu(menuName = "Auction/Stage/Randomised")]
+public class RandomisedAuctionStage : AuctionStage
```

Comment:
Aha, so you moved the randomized stage here. I guess we could keep it in its current incarnation for now... However, is it actually usable?